buildSrc/checksum.xml

```diff
@@ -26,7 +26,6 @@
     <trusted-key id='6449005f96bc97a3' group='de.undercouch' />
     <trusted-key id='379ce192d401ab61' group='info.picocli' />
     <trusted-key id='efe8086f9e93774e' group='junit' />
-    <trusted-key id='c84125c13bf6f2f2' group='org.ajoberstar.grgit' />
```

Comment:
Is there a reason you removed the PGP key reference?

Reply:
Thanks! I have reverted this line after reading [this](https://github.com/apache/jmeter/pull/488)